systemtest/test_content_length.py

```diff
@@ -103,16 +103,23 @@ def test_content_length_1000(self, posturl, crash_generator, nginx):
             with http_post(posturl, headers, payload) as resp:
                 status_code = resp.getcode()
         except RemoteDisconnected:
-            # If there's an ELB and nginx times out waiting for the rest of the
-            # request, then we get an HTTP 504. If there's no ELB (we're
-            # connecting directly to nginx), then nginx just drops the
-            # connection and we get back a RemoteDisconnected error.
-            status_code = 504
+            # If there's no LB (we're connecting directly to nginx), then nginx
+            # may drop the connection and we get back a RemoteDisconnected error
+            status_code = None
 
-        # Verify we get an HTTP 504 because something timed out waiting for the
+        # Verify we get an error because something timed out waiting for the
         # HTTP client (us) to send the rest of the data which is expected
         # because we sent a bad content-length
-        assert status_code == 504
+        assert status_code in (
+            # without LB, nginx drops the connection
+            None,
```

Comment:
What environment do we run the system tests in that doesn't have a load balancer, but does have nginx?

Reply:
when i have a spare moment: local dev

Reply:
I don't understand what that means. Does that mean you wrote up a bug to change the local dev environment such that it includes nginx running in local dev?

Reply:
good point, i have filed a bug about it https://bugzilla.mozilla.org/show_bug.cgi?id=1894955